geode-lucene/src/upgradeTest/java/org/apache/geode/cache/lucene/RollingUpgradeQueryReturnsCorrectResultsAfterClientAndServersAreRolledOverAllBucketsCreated.java

```diff
@@ -31,6 +32,7 @@
 public class RollingUpgradeQueryReturnsCorrectResultsAfterClientAndServersAreRolledOverAllBucketsCreated
     extends LuceneSearchWithRollingUpgradeDUnit {
 
+  @Ignore
```

Comment:
Don't commit this with `@Ignore` on a test.

Reply:
Upgrading to the Lucene 7.1.0 this test is not supported as formatting is changed between versions. In that case we add note to the documentation so all cluster members must be on the same version in order to execute Lucene queries.